### DISCUSSION TITLE
Hello, I'm trying to implement a plug-in for opening a door

### DISCUSSION THREAD

**Discussion by barashF** [SmartlyDressedGames/U3-Docs#261] [Q&A]
Faced the problem that after applying door.updateToggle(open); the door opens, but the texture remains in place. On the advice, I tried to implement it through BarricadeManager.ServerSetDoorOpen, but the library does not have this method. Please tell me what I'm doing wrong, thanks in advance!

**Comment by carlostipton1319** [comment]
As a user, implementing a plug-in for opening a [door tricks](https://topnewtechnology.com/the-ultimate-guide-to-garage-door-replacement-tips-and-tricks/ )can be a rewarding project. Depending on the specific context and platform you are working with, there are different approaches you can take.

**Reply by andriivitiv** [reply]
ChatGPT has entered the chat
